README.md

```diff
@@ -37,6 +37,51 @@ https://annu12340.github.io/Javascript-Notes/
 - JavaScript ES2020
 - Tricky JavaScript interview questions and answers
 
+##Infinities
+
+ Infinity is a property of the global object , or in other words, a variable in global scope. 
+ The initial value of Infinity is Number.POSITIVE_INFINITY . The JavaScript keyword Infinity 
+ points to the same internal value as POSITIVE_INFINITY. For practical purposes, "Infinity" can 
+ be used interchangeably with "Number.POSITIVE_INFINITY" in your scripts.
```

Comment:
I don't think this line is required
Let's keep the file as simple as possible

Reply:
Done